miniml/compiler/README.md

```diff
@@ -0,0 +1,16 @@
+# miniml compiler
+
+The file `compile.scm` is a single-file compiler for the miniml language, written in Scheme. It targets the OCaml bytecode for the version 4.07.0 of the OCaml compiler.
+
+It is used as such:
+```bash
+$ guile compile.scm file.ml -o file.byte
+```
+The file `file.byte` will then contain executable OCaml bytecode, which can be run using `ocamlrun` from version 4.07.0 of the OCaml compiler (in this repository's `ocaml-src/` submodule).
+
+It can only process a single file of input; in case you wish to use multiple files, you must first bundle them all into a single `.ml` file, putting the contents of file `x.ml` between `module X = struct`/`end` delimiters.
```

Comment:
(This is not a change proposal for the PR.) Actually it would be nice to lift this limitation, and probably not too hard, accepting several `.ml` files and interpreting each of them as if they were within a module of the corresponding name.